imports/ui/Review.jsx

```diff
@@ -21,6 +21,12 @@ export default class Review extends Component {
       liked: false, //indicates whether or not the review has been liked in the current state
     }
 
+    this.circlebox_class = props.isPreview ? "circlebox-preview" : "circlebox";
+    this.review_number_text_class = props.isPreview ? "review-number-text-preview" : "review-number-text";
+    this.review_number_label_class = props.isPreview ? "review-number-label-preview" : "review-number-label";
+    this.professor_title_class = props.isPreview ? "professor-title-preview" : "professor-title";
+    this.review_text_class = props.isPreview ? "review-text-preview" : "review-text";
+    this.professor_text_class = props.isPreview ? "professor-text-preview" : "professor-text";
```

Comment:
The code here looks dangerous. It relies on the assumption that `isPreview` prop never changes. Consider using functions to compute class based on the prop.
Also, camelCase for variables to be consistent

Reply:
`isPreview` never changes on this screen. I can discuss further at TPM meeting but for now we'll leave it like this.